### PR TITLE
add Travis and fix stuff

### LINKED ISSUE #183: LalkaChat degradation
with each new revision something falls off.  You need test...

### PR DESCRIPTION
**Pull request by JAremko**
Add  build tests(they are free and can work with forks. So it will help to make sure that a PR code works):
 - Test if we can build LalkaChat
 - Test if we can launch LalkaChat, connect to streaming platforms, bootstrap models without crashing.
 - Test if LalkaChat is properly started and available at http://localhost:8080/

https://travis-ci.org/JAremko/LalkaChat/builds/177396115

*We use pre-made config files to load all modules and connect to chats*

fixes #183  solves #184

### CODE REVIEW

**Comment by JAremko** [comment]
For now it's a very basic testing, but it can be upgraded to be godlike testing:
 1. Get Twitch channel and configure it to allow mods/owner only chat
 3. Add a thingy to the Travis tests that  will connect to the Twitch channel with a mod/owner account and to the LalkaChat websocket
 4. The thingy will:
  1.  Send a test message to the channel
  2.  Read it from the LalkaChat WebSocket
  3.  Fail the test if the messages don't match
 5. Repeat if for all streaming platforms

**Comment by JAremko** [comment]
https://travis-ci.org/JAremko/LalkaChat/builds/178433845#L582

File: docker/run [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
 else
-   printf '[server]\nhost = 0.0.0.0\nshow_browser = False\n' > $webchatCF
+   printf '[server]\nhost = 0.0.0.0\n\n' > $webchatCF
+fi
+
+if [ -f $globCF ];
+then
+   sed -i 's/^show_browser =.*$/show_browser = False/' $globCF
+else
+   printf '[gui]\nshow_browser = False\n\n' > $globCF
```
Comment (DeForce) [inline docker/run]: nitpicking, but please change to echo

File: docker/run [hunk 2]
```diff
@@ -1,13 +1,20 @@
...
 webchatCF="/usr/lib/python2.7/site-packages/LalkaChat/conf/webchat.cfg"
+globCF="/usr/lib/python2.7/site-packages/LalkaChat/conf/config.cfg"
 
 if [ -f $webchatCF ];
 then
    sed -i 's/^host =.*$/host = 0.0.0.0/' $webchatCF
-   sed -i 's/^show_browser =.*$/show_browser = False/' $webchatCF
 else
-   printf '[server]\nhost = 0.0.0.0\nshow_browser = False\n' > $webchatCF
+   printf '[server]\nhost = 0.0.0.0\n\n' > $webchatCF
```
Comment (DeForce) [inline docker/run]: cat << EOF | tee > ${webchatCF}
[server]
host = 0.0.0.0

EOF

File: docker/run (hunk 1 above)
Comment (DeForce) [inline docker/run]: or change to cat > tee which is better

**Pull request merged**